src/helpers/configHelpers.ts

```diff
@@ -158,7 +158,7 @@ export function deleteConfigsFromUnknownServers(client: Client): void {
 	});
 }
 
-function readConfigFromFile(guildId: string): NeedleConfig | undefined {
+export function readConfigFromFile(guildId: string): NeedleConfig | undefined {
```

Comment:
Why aren't we using [`getConfig()`](https://github.com/c43721/discord-needle/blob/4e8901f0b8eadb33cbfd30d8d6d1faaa72b3e1c4/src/helpers/configHelpers.ts#L26) for this? I don't think this should be exposed.

Reply:
couldn't find that method 😐